src/appshell/qml/DevTools/Gallery/GeneralComponentsGallery.qml

```diff
@@ -28,6 +28,7 @@ import MuseScore.Ui 1.0
 
 Rectangle {
     id: root
+    clip: true
```

Comment:
Please move this line under the color property
According to our qml code style

Reply:
Done